analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/HashesShouldHaveUnpredictableSalt.vb

```diff
@@ -0,0 +1,235 @@
+﻿Imports System
+Imports System.Security.Cryptography
+Imports System.Text
+Imports AliasedPasswordDeriveBytes = System.Security.Cryptography.PasswordDeriveBytes
+
+Class Program
+    Private Const passwordString As String = "Secret"
+    Private cspParams As CspParameters = New CspParameters()
+    Private ReadOnly passwordBytes As Byte() = Encoding.UTF8.GetBytes(passwordString)
+
+    ' Out of the two issues (salt is too short vs. salt is predictable) being predictable is the more serious one.
+    ' If both issues are present then the rule's message will reflect on the salt being predictable.
+    Public Sub ShortAndConstantSaltIsNotCompliant()
+        Dim shortAndConstantSalt = New Byte(14) {}
+        Dim pdb1 = New PasswordDeriveBytes(passwordBytes, shortAndConstantSalt)                                                      ' Noncompliant {{Make this salt unpredictable.}}
```

Comment:
Could you please add an object creation with all capitals - as VB.NEt is case insensitive?
There should be no issue but better be sure.

Reply:
Good catch! It got removed by auto-formatting 😃